src/library/scala/collection/mutable/BitSet.scala

```diff
@@ -170,6 +183,60 @@ class BitSet(protected[collection] final var elems: Array[Long])
   override def zip[B](that: IterableOnce[B])(implicit @implicitNotFound(collection.BitSet.zipOrdMsg) ev: Ordering[(Int, B)]): SortedSet[(Int, B)] =
     super.zip(that)
 
+  def oldAddAll(xs: IterableOnce[Int]): this.type = xs match {
+    case bs: collection.BitSet => this |= bs
+    case _ => super.addAll(xs)
+  }
+
+  override def addAll(xs: IterableOnce[Int]): this.type = xs match {
+    case bs: collection.BitSet => this |= bs
+    case range: Range if (range.step == 1 || range.step == -1) =>
+      val start = range.min
+      if (start >= 0) {
+        val startIdx = start >> LogWL
+        val end = range.max
+          val endIdx = end >> LogWL
```

Comment:
indentation appears to have gotten messed up here

Reply:
Fixed